packages/next/src/build/webpack-config.ts

```diff
@@ -696,7 +696,7 @@ export default async function getBaseWebpackConfig(
     // Disable .mjs for node_modules bundling
     extensions: config.experimental?.resolveExtensions
       ? config.experimental.resolveExtensions
-      : ['.js', '.mjs', '.tsx', '.ts', '.jsx', '.json', '.wasm'],
+      : ['.js', '.mjs', '.tsx', '.ts', '.jsx', '.json', '.node'],
```

Comment:
[Turbopack's default value](https://github.com/vercel/turbo/blob/main/crates/turbopack-resolve/src/resolve.rs#L175-L199) has the same extensions but in a different order: `.tsx, .ts, .jsx, .js, .mjs, .node, .json`. I could reorder one so they match, but I guess there is a reason why Turbopack was set with a different order.